src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "[X]":"[ ]");
+    }
+
+    public void updateDoneStatus(){
+        isDone = true;
+    }
+
+    public String getDateTime() {
+        return null;
+    }
+
+    public String getTaskType() {
```

Comment:
Maybe we can make the task type a protected attribute instead of overriding the return value of this method?